koku/api/settings/settings.py

```diff
@@ -112,40 +102,22 @@ def __init__(self, request):
     def _get_tag_management_prefix(self, providerName):
         return f"{SETTINGS_PREFIX}.tag-management.{providerName}"
 
-    def _obtain_tag_keys(self, tag_view_kls, tag_handler_kls, tag_keys_kls):
+    def _obtain_tag_keys(self, tag_keys_kls):
         """
         Collect the available tag keys for the customer.
 
         Returns:
             (List) - List of available tag keys objects
             (List) - List of enabled tag keys strings
         """
-        url = (
-            "?filter[time_scope_units]=month&filter[time_scope_value]=-1"
-            "&filter[resolution]=monthly&key_only=True&filter[enabled]=False"
-        )
-        tag_request = self.factory.get(url)
-        tag_request.user = self.request.user
-        query_params = QueryParameters(tag_request, tag_view_kls)
-        handler = tag_handler_kls(query_params)
-        query_output = handler.execute_query()
-        avail_data = query_output.get("data")
-        all_tags_set = set(avail_data)
         enabled = []
+        all_tags_set = set()
         with schema_context(self.schema):
-            if tag_keys_kls in (AWSEnabledTagKeys, AzureEnabledTagKeys):
-                all_tags_set = set()
-                for tag_key in tag_keys_kls.objects.all():
-                    all_tags_set.add(tag_key.key)
-                    if tag_key.enabled:
-                        enabled.append(tag_key.key)
-            else:
-                enabled_tags = tag_keys_kls.objects.all()
-                enabled = [enabled_tag.key for enabled_tag in enabled_tags]
-
-            all_tags_set.update(enabled)
-
-        return all_tags_set, enabled
+            for tag_key in tag_keys_kls.objects.all():
```

Comment:
Is it possible to sort via Django here?

Reply:
Discussed and switched to using set logic.